src/hotspot/share/c1/c1_Runtime1.cpp

```diff
@@ -659,89 +659,87 @@ address Runtime1::exception_handler_for_pc(JavaThread* thread) {
 }
 
 
-JRT_ENTRY(void, Runtime1::throw_range_check_exception(JavaThread* thread, int index, arrayOopDesc* a))
+JRT_ENTRY(void, Runtime1::throw_range_check_exception(JavaThread* current, int index, arrayOopDesc* a))
   NOT_PRODUCT(_throw_range_check_exception_count++;)
   const int len = 35;
   assert(len < strlen("Index %d out of bounds for length %d"), "Must allocate more space for message.");
   char message[2 * jintAsStringSize + len];
   sprintf(message, "Index %d out of bounds for length %d", index, a->length());
-  SharedRuntime::throw_and_post_jvmti_exception(thread, vmSymbols::java_lang_ArrayIndexOutOfBoundsException(), message);
+  SharedRuntime::throw_and_post_jvmti_exception(current, vmSymbols::java_lang_ArrayIndexOutOfBoundsException(), message);
 JRT_END
 
 
-JRT_ENTRY(void, Runtime1::throw_index_exception(JavaThread* thread, int index))
+JRT_ENTRY(void, Runtime1::throw_index_exception(JavaThread* current, int index))
   NOT_PRODUCT(_throw_index_exception_count++;)
   char message[16];
   sprintf(message, "%d", index);
-  SharedRuntime::throw_and_post_jvmti_exception(thread, vmSymbols::java_lang_IndexOutOfBoundsException(), message);
+  SharedRuntime::throw_and_post_jvmti_exception(current, vmSymbols::java_lang_IndexOutOfBoundsException(), message);
 JRT_END
 
 
-JRT_ENTRY(void, Runtime1::throw_div0_exception(JavaThread* thread))
+JRT_ENTRY(void, Runtime1::throw_div0_exception(JavaThread* current))
   NOT_PRODUCT(_throw_div0_exception_count++;)
-  SharedRuntime::throw_and_post_jvmti_exception(thread, vmSymbols::java_lang_ArithmeticException(), "/ by zero");
+  SharedRuntime::throw_and_post_jvmti_exception(current, vmSymbols::java_lang_ArithmeticException(), "/ by zero");
 JRT_END
 
 
-JRT_ENTRY(void, Runtime1::throw_null_pointer_exception(JavaThread* thread))
+JRT_ENTRY(void, Runtime1::throw_null_pointer_exception(JavaThread* current))
   NOT_PRODUCT(_throw_null_pointer_exception_count++;)
-  SharedRuntime::throw_and_post_jvmti_exception(thread, vmSymbols::java_lang_NullPointerException());
+  SharedRuntime::throw_and_post_jvmti_exception(current, vmSymbols::java_lang_NullPointerException());
 JRT_END
 
 
-JRT_ENTRY(void, Runtime1::throw_class_cast_exception(JavaThread* thread, oopDesc* object))
+JRT_ENTRY(void, Runtime1::throw_class_cast_exception(JavaThread* current, oopDesc* object))
   NOT_PRODUCT(_throw_class_cast_exception_count++;)
-  ResourceMark rm(thread);
-  char* message = SharedRuntime::generate_class_cast_message(
-    thread, object->klass());
-  SharedRuntime::throw_and_post_jvmti_exception(
-    thread, vmSymbols::java_lang_ClassCastException(), message);
+  ResourceMark rm(current);
+char* message = SharedRuntime::generate_class_cast_message(current, object->klass());
```

Comment:
Is this indentation off?

Reply:
Fixed. (My emacs can't figure out how to indent when these macros are used. :( )